### ISSUE TITLE
AWS EC2 Linux 2017.03 Support

### ISSUE THREAD

**Issue by rrdoffin** [nodesource/distributions#1268]
It seems like just this past week, I was able to install nodejs v4. Now, when I attempt it, the install script informs me that this version is no longer support. And, it recommends several other more current versions like v12, v14, or v16. I attempted to install v14, but it would not install and displayed the following message:

 Your distribution, identified as "system-release-2017.03-0.0.noarch", is not currently supported, please contact NodeSource at https://github.com/nodesource/distributions/issues if you think this is incorrect or would like your distribution to be considered for support

Can support be added for this distribution?

**Comment by ianwremmel** [comment]
@rrdoffin see https://github.com/nodesource/distributions/issues/1266

Tl;DR: The problem isn't a missing package; your host can't make an SSL handshake to download node because the original Let's Encrypt root certificate expired at midnight UTC today. You need to get the latest Let's Encrypt root certificate onto your host.

**Comment by pulkit21** [comment]
@rrdoffin please try this `https://aws.amazon.com/premiumsupport/knowledge-center/ec2-expired-certificate/` mine got fixed

**Comment by rrdoffin** [comment]
The issue is resolved. Thanks for the tips on updating the CA Cert.
